install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -8,6 +8,12 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: istio-pilot
 rules:
+- apiGroups: ["config.istio.io"]
+  resources: ["istiokinds"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
```

Comment:
I figured that was probably the case. I guess we can reconsider splitting that out later once new Istio CRD types are stabilized.